py/client/pydeephaven/table.py

```diff
@@ -219,6 +220,23 @@ def sort(self, order_by: Union[str, List[str]], order: Union[SortDirection, List
         """
         return super().sort(order_by, order)
 
+    def sort_descending(self, order_by: Union[str, List[str]]) -> Table:
+        """The sort_descending method creates a new table where rows in a table are sorted in a largest to smallest
```

Comment:
Is largest to smallest the right way to phrase this? Descending order, or "last to first" or something might be better.

Reply:
```suggestion
        """The sort_descending method creates a new table where rows in a table are sorted in descending
```
I think this is most consistent with our `sort` docs.